Resources/config/doctrine.xml

```diff
@@ -12,6 +12,8 @@
             <argument>%fos_user.model.user.class%</argument>
         </service>
 
+        <service id="FOS\UserBundle\Model\UserManagerInterface" alias="fos_user.user_manager" public="false" />
```

Comment:
this would not make it work for people not using Doctrine. We must add it in a file being always loaded

Reply:
In which file should we add it ? A custom new one ?